apps/omg_performance/test/omg_performance/byzantine_event_test.exs

```diff
@@ -62,7 +73,26 @@ defmodule OMG.Performance.ByzantineEventsTest do
 
     :ok = ByzantineEvents.watcher_synchronize(root_chain_height: last_exit_height)
     # assert that we can call this testing function reliably and that there are no invalid exits
-    assert [] = ByzantineEvents.get_byzantine_events("invalid_exit")
+    assert [] = ByzantineEvents.get_byzantine_events()
+  end
+
+  @tag fixtures: [:perf_test, :mix_based_child_chain, :mix_based_watcher]
+  test "can provide timing of status.get under many valid IFEs for included txs",
```

Comment:
what does the test trying to prove? 
`can provide timing of status.get under many valid IFEs for included txs`

Reply:
see earlier comment, if we decide on the wording, I'll reword consistently